test/Microsoft.Azure.Devices.Edge.Test.TempSensor/Test.cs

```diff
@@ -6,100 +6,126 @@ namespace Microsoft.Azure.Devices.Edge.Test.TempSensor
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Edge.Test.Common;
     using Microsoft.Azure.Devices.Edge.Util;
+    using Serilog;
+    using Serilog.Events;
 
     public class Test
     {
-        public Task<int> RunAsync(Args args) => Profiler.Run(
-            "Running tempSensor test",
-            async () =>
+        public async Task<int> RunAsync(Args args)
+        {
+            LogEventLevel consoleLevel = args.Verbose
+                ? LogEventLevel.Verbose
```

Comment:
Should this be in a more common location (Test.Common instead of TempSensor)

Reply:
I mean, we can parse out the options and init in some generic place. It feels like all other modules will also need the same code.

Reply:
Yes I expect more code to move from temp sensor to common, especially when I add more tests. I don't want each test to have it's own log level mapping, but I want to wait for another test before I decide how to do it.